src/Mod/PartDesign/Gui/ViewProviderBoolean.cpp

```diff
@@ -120,11 +120,15 @@ bool ViewProviderBoolean::onDelete(const std::vector<std::string> &s)
     return ViewProvider::onDelete(s);
 }
 
-void ViewProviderBoolean::attach(App::DocumentObject* obj) {
+void ViewProviderBoolean::attach(App::DocumentObject* obj)
+{
     PartGui::ViewProviderPartExt::attach(obj);
+}
 
+const char* ViewProviderBoolean::getDefaultDisplayMode() const
+{
     //set default display mode to override the "Group" display mode

```

Comment:
This comment is out of context and not needed anymore. It should be clear enough that the default mode is "Flat Lines" since this is the function that defines the default display mode, Right?
```suggestion
```